kubernetes_state/metadata.csv

```diff
@@ -27,6 +28,7 @@ kubernetes_state.deployment.rollingupdate.max_unavailable,gauge,,,,Maximum numbe
 kubernetes_state.endpoint.address_available,gauge,,,,Number of addresses available in endpoint,0,kubernetes,k8s_state.endpoint.address_avail
 kubernetes_state.endpoint.address_not_ready,gauge,,,,Number of addresses not ready in endpoint,-1,kubernetes,k8s_state.endpoint.address_not_ready
 kubernetes_state.endpoint.created,gauge,,,,Unix creation timestamp,0,kubernetes,k8s_state.endpoint.creation_time
+kubernetes_state.job,count,,,,The number of jobs,0,kubernetes,k8s_state.job.count
```

Comment:
this one is a count but the others are gauges? 🤔

Reply:
You are right, this should be a count gauge as well I think 👍